packages/sidecar/src/formatters/logfmt/errors.ts

```diff
@@ -0,0 +1,71 @@
+import type { ErrorEvent } from "@sentry/core";
+import logfmt from "logfmt";
+
+export function formatError(event: ErrorEvent): string {
+  const data: Record<string, any> = {
+    type: "error",
+    event_id: event.event_id,
+    timestamp: event.timestamp ? new Date(event.timestamp * 1000).toISOString() : new Date().toISOString(),
```

Comment:
I think we should use a centralized helper function to avoid repeating this logic around